docs/source/user_guide/input_output.md

```diff
@@ -319,13 +347,78 @@ with open(filepath, mode="w", newline="") as file:
 ```
 Alternatively, we can convert the `movement` bounding boxes dataset to a pandas DataFrame with the {meth}`xarray.DataArray.to_dataframe` method, wrangle the dataframe as required, and then apply the {meth}`pandas.DataFrame.to_csv` method to save the data as a .csv file.
 
+(target-netcdf)=
+## Native saving and loading with netCDF
+
+Because `movement` datasets are {class}`xarray.Dataset` objects, we can rely on
+xarray's built-in [support for the netCDF file format](xarray:user-guide/io.html).
+
+netCDF is a binary file format for self-described datasets that originated in the geosciences,
+and netCDF files on disk directly correspond to {class}`xarray.Dataset` objects.
+
+Saving to netCDF is the recommended way to preserve the complete state of your analysis,
+including all variables, coordinates, and attributes.
+
+To save any xarray dataset `ds` to a netCDF file:
+
+```python
+ds.to_netcdf("/path/to/my_data.nc")
+````
+
+To load the dataset back:
+
+```python
+import xarray as xr
+
+ds = xr.open_dataset("my_data.nc")
+```
+
+Similarly, an {class}`xarray.DataArray` object (e.g. the `position` variable
+of a `movement` dataset) can be saved to disk using the
+{meth}`to_netcdf()<xarray.DataArray.to_netcdf()>` method, and loaded from disk using the
+{func}`xarray.open_dataarray()` function.
+As netCDF files correspond to Dataset objects,
+these functions internally convert the DataArray to a Dataset before saving,
+and then convert back when loading.
+
+:::{note}
+xarray also supports compression and chunking options with netCDF, which can be useful for managing large datasets.
+For more details, see the [xarray documentation on netCDF](xarray:user-guide/io.html).
+:::
+
+Below is a realistic example of how you may integrate netCDF into you
```

Comment:
For some reason I don't like the specification of the example being "realistic". Are all the other examples unrealistic?
```suggestion
Below is an example of how you may integrate netCDF into you
```